### PR TITLE
koji-hub crd docs

### LINKED ISSUE #28: koji-hub crd documentation
As an User, I want to know which properties can be set in the koji-hub CR.

## Acceptance Criteria

* A `docs` folder in the root project dir containing a README file on the CRD usage;
* Document each property name, value type and  short description on its purpose.

## Definition of Done

Sphinx html docs that contain the explanation of this custom resource and its properties.

### PR DESCRIPTION
**Pull request by odra**
Fixes #28

## Changes

* Adds koji-hub crd sphinx docs

##  Verification
* cd into `docs`
* run `make html`
* access the docs: `docs/build/html/index.html` and a user guide/koji-hub section should be available

### CODE REVIEW

File: docs/source/user-guide/koji-hub.rst [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+------------
+
+The fedora messaging topic prefix to use koji-hub config.
+
+fedora_messaging_url
+--------------------
+
+The fedora messaging url to use in koji-hub.
+
+This isa required property with no default value.
```
Comment (Zlopez) [inline docs/source/user-guide/koji-hub.rst:184]:
```suggestion
This is a required property with no default value.
```

File: docs/source/user-guide/koji-hub.rst [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+A config map that contains fedora messaging certs to be mounted in koji-hub pod filesystem.
+
+Those files are used to authenticate koji-hub to a fedora-messaging instance.
+
+Config map fomrat:
+
+.. code-block:: yaml
+
+  apiVersion: v1
+  kind: ConfigMap
```
Comment (Zlopez) [inline docs/source/user-guide/koji-hub.rst:198]: This looks like secret to me

File: docs/source/user-guide/koji-hub.rst [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+The kubernetes ingress backend to use when creating an ingress resource for koji-hub.
+
+Available choices:
+
+* nginx
+
+Usage
+=====
+
+Create a file containing the following content (modify as needed):
```
Comment (Zlopez) [inline docs/source/user-guide/koji-hub.rst:225]: We could also add link to existing file in repo.

File: docs/source/user-guide/koji-hub.rst (hunk 2 above)
Comment (odra) [reply docs/source/user-guide/koji-hub.rst:198]: it is a configmap now, I will file an issue to make it a secret.

**Pull request merged**